organization.go

```diff
@@ -24,10 +25,13 @@ type OrganizationChild struct {
 	ID   int64  `json:"id"`
 }
 
-func (c *Client) ListOrganization(minimal bool) ([]*Organization, error) {
+func (c *Client) ListOrganization(filter map[string]string) ([]*Organization, error) {
 	res := make([]*Organization, 0, 5)
-	path := fmt.Sprintf("api/v2/organization?minimal=%t", minimal)
-	req := c.newRequest(http.MethodGet, path, nil)
+	query := url.Values{}
```

Comment:
The filter loop is used in many places now. Perhaps a new Type instead of map[string]string can be created with a function ToURLValues()?